### ISSUE TITLE
Feature request: Add option for font-display property

### ISSUE THREAD

**Issue by andycochrane** [oddbird/accoutrement-type#10]
There isn't a way to add this property currently and I can't see a way around it.

Self-explanatory request, but let me know if you need any more info and I'll be happy to update this.

Thanks.

**Comment by mirisuzanne** [comment]
Yes, good. This raises another issue: we should have a way of supporting arbitrary properties, so we don't have to update every time a new property is available. It makes sense for weight/style to be built in, as they tend to map directly with imported files – but properties like `font-display` do not need any automation.
